app/templates/src/handlebars/helpers/helpers.js

```diff
@@ -15,4 +15,8 @@ function addYear(s) {
 	return s + ' ' + new Date().getFullYear();
 }
 
-module.exports = addYear;
+module.exports = {
+	/* stylelint-disable */ // because handlebars isn't able to work with shorthands
```

Comment:
@mischah can you take a look at this? Maybe you know a solution for handlebars complaining about missing helpers if exported using shorthand property syntax.